Alexa.NET/Response/Response.cs

```diff
@@ -14,8 +16,30 @@ public class ResponseBody
         [JsonProperty("reprompt", NullValueHandling = NullValueHandling.Ignore)]
         public Reprompt Reprompt { get; set; }
 
+        private bool? _shouldEndSession = null;
+
         [JsonProperty("shouldEndSession", NullValueHandling = NullValueHandling.Ignore)]
-        public bool? ShouldEndSession { get; set; } = false;
```

Comment:
This changes effectively now changes a default behavior where ShouldEndSession was previous false as default and now is null as default.  Should we retain the default as false and _shouldEndSession be set to false?

Reply:
Absolutely - existing behavior shouldn't be altered unless a new directive is used.
Updated accordingly :+1: